Makefile

```diff
@@ -205,4 +205,15 @@ check-deps: ## Check if there are any unused dependencies in Cargo.toml
 
 .PHONY: test-workflow
 test-workflow: ## Tests a github actions workflow locally using `act`
-	@act -W ./.github/workflows/$(file)
\ No newline at end of file
+	@act -W ./.github/workflows/$(file)
+
+.PHONY: stage-release-artifacts
+stage-release-artifacts: ## Stage release artifacts
+	@mkdir -p /tmp/release
+	@cp ~/.cargo/bin/auraed /tmp/release/auraed-$(tag)-$(uname_m)-unknown-linux-musl
+	@cp ~/.cargo/bin/auraescript /tmp/release/auraescript-$(tag)-$(uname_m)-unknown-linux-musl
```

Comment:
Does auraescript compile with musl for you? `cargo build -p auraescript --target aarch64-unknown-linux-musl` fails to compile for me.

Reply:
No, looks like I got an error too. I ended up grabbing a used Xeon workstation to throw a dev environment on, so I've been using that more.

Reply:
Does it work on x86? I thought it wouldn't work at all with musl

Reply:
Yea it seems to have. I ran 'make release' and had an auraescript binary that worked as an output.

Reply:
Just checked, `make release` doesn't use musl for auraescript (it uses the default which will be gnu)